README.md

```diff
@@ -121,55 +129,25 @@ https://github.com/WebAssembly/esm-integration/issues/56.
 This property does not just impact platforms using CSP, but also other platforms
 with systems to restrict permissions, such as Deno.
 
-### Module Blocks
-
-Ideally this proposal would use the module object defined by the [module blocks][] proposal.
-
-Just as module blocks permit the dynamic import of blocks relating to their primary host instance,
-it could be worth supporting the same for `WebAssembly.Module` for symmetry under a reflection model:
-
-```js
-import module WasmModule from './module.wasm';
-
-WasmModule instanceof WebAssembly.Module; // true
-
-await import(WasmModule); // returns the fully linked and instantiated ModuleNamespace
-```
-
-It is currently not clear if this would provide any strong benefits, therefore it is not currently specified, but could be an option for additions to this specification.
-
-Implementing the above would require new specification machinery to relate a special internal slot on the `WebAssembly.Module`
-to its Cyclic Module Record. One benefit of this approach would also be enabling Wasm modules to participate in cycles with JS modules.
-
-### Compartments
-
-Import reflection aims to be compatible with compartments hooks.
-
-Compartment hooks may benefit from the ability to securely import module records as discussed in the security section.
-
-This would allow creating custom loaders without requiring relaxing the security guarantees of the environment
-since all compiled modules being loaded can be fully audited by the host instead of having to enable arbitrary modular evaluation string sources.
-
-### Workers
-
-Since `WebAssembly.Module` and the `Module` object in module blocks both support being passed to a worker, we should with this alignment have all reflections being worker-compatible.
-
 ## Cache Key Semantics
 
-Semantically this proposal involves a relaxation of the `HostResolveImportedModule` idempotency requirement.
+Semantically this proposal involves a relaxation of the
```

Comment:
I don’t interpret this as a relaxation of idempotence requirements. Every existing dependency remains as idempotent as it was, but this introduces a parallel memo for module reflections on the same key. This does of course generalize to an entry for every (reflection type, full module specifier), but even then, the columnar interpretation (a new memo for each interpretation of the asset bytes), makes clear that no invariants regress.

Reply:
I've removed this line.